rj_gameplay/rj_gameplay/play/defensive_clear.py

```diff
@@ -0,0 +1,50 @@
+import stp.play as play
+import stp.tactic as tactic
+
+from rj_gameplay.tactic import stub_striker, nmark_tactic
+import stp.skill as skill
+import stp.role as role
+from stp.role.assignment.naive import NaiveRoleAssignment
+import stp.rc as rc
+from typing import Dict, Generic, Iterator, List, Optional, Tuple, Type, TypeVar
+
+class DefensiveClear(play.IPlay):
+
+    def __init__(self):
+        # TODO: add chipper tactic here
+
+        self.two_mark = nmark_tactic.NMarkTactic(2)
+        self.role_assigner = NaiveRoleAssignment()
+
+    def compute_props(self, prev_props):
+    	pass
+
+    def tick(
+        self,
+        world_state: rc.WorldState,
+        prev_results: role.assignment.FlatRoleResults,
+        props,
+    ) -> Tuple[Dict[Type[tactic.SkillEntry], List[role.RoleRequest]], List[tactic.SkillEntry]]:
+
+    	# Get role requests from all tactics and put them into a dictionary
+    	role_requests: play.RoleRequests = {}
+    	# role_requests[self.striker_tactic] = self.striker_tactic.get_requests(world_state, None)
+    	role_requests[self.two_mark] = (self.two_mark.get_requests(world_state, None))
```

Comment:
Shouldn't this have a tactic for the robot kicking the ball? And also in theory a pass seeking tactic.

Reply:
Yes and yes. I started off writing the defensive clear play only to realize both the nmark and striker tactics weren't written yet. So this PR is step 1. For step 2 I'll write a striker tactic, uncomment the aforementioned "dead code", and possibly add the pass seeker, then put up a new PR for the full defensive clear play.